src/particle/particle/particle.py

```diff
@@ -427,7 +429,7 @@ def to_list(
 
     @classmethod
     def to_dict(cls, *args, **kwargs):
-        # type: (Any, Any) -> Dict[str, List[Union[str, float, bool, int]]]
+        # type: (Any, Any) -> Dict[str, Tuple[Union[bool, int, str, float], ...]]
```

Comment:
This returns a tuple, not a list.

Reply:
And the key is a string, it's the value that's a Tuple.

Reply:
I had a Tuple before in my many trials but the hints were tricking me. For sure `str, Tuple` makes sense. I was definitely missing the `...`. Thanks a lot!